esmvaltool/recipes/recipe_seaice.yml

```diff
@@ -0,0 +1,151 @@
+# ESMValTool
+# recipe_seaice.yml
+---
+documentation:
+  description: |
+    Recipe for plotting sea ice diagnostics at the Arctic and Antarctic.
+
+  authors:
+    - senf_da
+
+  maintainer:
+    - senf_da
```

Comment:
I would name someone from the core team as maintainer (e.g., righ_ma or laue_ax).